src/logger.ts

```diff
@@ -0,0 +1,97 @@
+import { Logger, LoggerModes } from '@overnightjs/logger'
+import morgan from 'morgan'
+import { config } from 'node-config-ts'
+import * as rfs from 'rotating-file-stream'
+
+enum LogLevel {
+  debug,
+  important,
+  warn
+}
+
+const logLevelMapping = {
+  'debug': LogLevel.debug,
+  'important': LogLevel.important,
+  'warn': LogLevel.warn
+}
+
+const LOG_LEVEL = logLevelMapping[config.logger.level] || LogLevel.important
+const LOG_TO_FILES = config.logger.logToFiles || false
+const LOG_PATH = config.logger.filePath || '/usr/local/var/log/cas'
+const METRICS_LOG_PATH = config.logger.metricsFilePath || LOG_PATH + '/metrics'
```

Comment:
So, if we're logging api calls to this file, then what we're logging isn't really metrics, but events.  We aggregate those events into metrics, but that happens at a higher level outside of the CAS.  So I think this should probably be call `eventsFilePath`.  In the future, we may want to be able to manually log events here beyond just those auto-generated by calls to the API endpoints.

Reply:
I think my other comment about access.log vs metrics/out.log clarifies

Reply:
Hmm... hard to say without seeing a concrete example, but I suspect even then we wouldn't actually be logging metrics?  Are you imagining log entries would include a field and it's *current* value?  Or will the log messages contain messages that result in a delta to a value that we aggregate on the other end?

Reply:
not sure yet